orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -127,7 +127,10 @@ def __init__(self, module, debug_options: DebugOptions, fallback_manager: _Fallb
         # List of opcodes to be considered safe to move before/after cast operation if propagate_cast_ops_level is zero.
         self._propagate_cast_ops_allow = []
         # Whether allow fusion of layer norm subgraph if doing so will cause modified precision.
-        self._allow_layer_norm_mod_precision = False
+        # This option is used to control SimplifiedLayerNormalization fusion when there is Cast Ops in the sub-graph
+        # for mix-precision training. LayerNorm kernels will use higher precision data type for inside computation
+        # so it's save to fuse to SimplifiedLayerNormalization for better perf.
```

Comment:
do you know why this was set to False before?

Reply:
I am guessing the author that time just used the default value for this flag and no one reported any effect to any model at that time?